src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -75,15 +77,35 @@ private Router setupRouter() {
               .end("Quote text must be provided in payload");
           }
   
-          return insertQuote(jsonObject.getString("author", "Unknown"), jsonObject.getString("text"))
+          return insertQuoteAndReturn(jsonObject.getString("author", "Unknown"), jsonObject.getString("text"))
               .compose(rows -> {
+                JsonObject insertedQuote = rows.iterator().next().toJson();
+
+                ctx.vertx()
+                  .eventBus()
+                  .publish(REALTIME_EVENT_ADDRESS, insertedQuote, new DeliveryOptions().setLocalOnly(true));
+              
                 return ctx.response()
                   .setStatusCode(HttpResponseStatus.CREATED.code())
-                  .end();
+                  .putHeader(HttpHeaderNames.CONTENT_TYPE, "application/json")
+                  .end(insertedQuote.encodePrettily());
               });
         }).onFailure(cause -> ctx.fail(cause));
     });
 
+    router.get("/realtime").handler(ctx -> {
+      ctx.request()
+        .toWebSocket()
+        .onSuccess(webSocket -> {
+          ctx.vertx()
+            .eventBus()
+            .localConsumer(REALTIME_EVENT_ADDRESS, message -> {
```

Comment:
Can you update this so that the local consumer is unregistered when the websocket gets closed?

Reply:
Updated it now. 
I also noticed in the document "Every WebSocket automatically registers two handler on the event bus, and when any data are received in this handler, it writes them to itself ".

Is it necessary to unregister those as well?

Reply:
No, if you the `binaryHandlerID` or the `textHandlerID` of the websocket, Vert.x unregister the local consumer automatically when the websocket is closed